Postal/IMAPSession+List.swift

```diff
@@ -29,7 +29,7 @@ extension IMAPSession {
     
     func listFolders() throws -> [Folder] {
         let prefix = defaultNamespace?.items.first?.prefix ?? MAIL_DIR_SEPARATOR_S
-        var list: UnsafeMutablePointer<clist> = nil;
+        var list: UnsafeMutablePointer<clist>? = nil
```

Comment:
Yes. That was the main problem I had during the migration. More information there: https://github.com/apple/swift-evolution/blob/master/proposals/0055-optional-unsafe-pointers.md